[core]/es_extended/server/main.lua

```diff
@@ -330,6 +330,14 @@ function loadESXPlayer(identifier, playerId, isNew)
     xPlayer.triggerEvent('esx:createMissingPickups', Core.Pickups)
   else
     exports.ox_inventory:setPlayerInventory(xPlayer, userData.inventory)
+
+    if isNew then
+      for account, money in pairs(Config.StartingAccountMoney) do
+        if account == 'money' or account == 'black_money' then
+          exports.ox_inventory:AddItem(playerId, account, money)
+        end
+      end
+    end
```

Comment:
If you don't use ox_inventory it will throw an error on the export, so this is wrong. I'll mark where it should be inserted to avoid writing more IFs.

Reply:
215 row under.

![image](https://github.com/esx-framework/esx_core/assets/22717950/05b25869-9e94-4bfa-b208-6fd5461fea83)

Reply:
I think if you make the above changes we can accept it afterwards.

Reply:
I dont really get what you mean. Because this is basicly already below an else where before if 
`if not Config.OxInventory then` is stated? in the same block also a export for ox inventory for setting the inventory and before that I can not add Items to the players inventory

![Screenshot_101](https://github.com/esx-framework/esx_core/assets/62698394/6c3710a3-a650-422c-b09b-fb1286b4d6f8)

Reply:
Yes my mistake because github hide the unmodified parts so it's correct how you did it sorry.